schema/definitions/0.8.0/examples/table_unsmry.yml

```diff
@@ -0,0 +1,175 @@
+  - datetime: 2020-10-28T14:46:14
+    user: 
+      id: peesv
+    event: updated
+
+class: table # class is the main identifier of the data type.
+
+fmu: # the fmu-block contains information directly related to the FMU context
+  model:
+    name: ff
+    revision: 21.0.0.dev
+    description:
+      - detailed description
+      - optional
+
+  workflow:
+    reference: simulation  # not sure about this one
+
+  case:
+    name: MyCaseName
+    uuid: 8bb56d60-8758-481a-89a4-6bac8561d38e
+    user:
+      id: jriv # $USER from ERT
+    description:
+      - yet other detailed description
+      - optional
+    restart_from: 15ce3b84-766f-4c93-9050-b154861f9100   # fmu.iteration.uuid for another iteration/case
+
+  iteration:
+    id: 0 # always an int, will be 0 for e.g. "pred"
+    uuid: 4b939310-34b1-4179-802c-49460bc0f799 # hash of case.uuid + iteration.id
+    name: "iter-0" # /"pred"
+
+  realization:
+    id: 33
+    uuid: 29a15b21-ce13-471b-9a4a-0f791552aa51
+    name: "realization-33"
+    parameters: # directly pass parameters.txt. This is potentially a lot of content, only a stub is included here.
+      SENSNAME: faultseal
+      SENSCASE: low
+      RMS_SEED: 1006
+      INIT_FILES:
+        PERM_FLUVCHAN_E1_NORM: 0.748433
+        PERM_FLUVCHAN_E21_NORM: 0.782068
+      KVKH_CHANNEL: 0.6
+      KVKH_US: 0.6
+      FAULT_SEAL_SCALING: 0.1
+      FWL_CENTRAL: 1677
+
+file:
+  relative_path: realization-33/iter-0/share/results/table/mysmry.arrow # case-relative
+  absolute_path: /some/absolute/path/realization-33/iter-0/share/results/table/mysmry.arrow
+  checksum_md5: kjhsdfvsdlfk23knerknvk23  # checksum of the file, not the data.
+  size_bytes: 5010144
+
+data: # The data block describes the actual data (e.g. surface). Only present in data objects
+
+  content: unsmry
+
+  # if stratigraphic, name must match the strat column. This is the official name of this surface.
+  name: myunsmry
+  stratigraphic: false
+
+  properties:
+  - name: DATE
+  - name: YEARS
+    unit: YEARS
+    is_total: false
+    is_rate: false
+    is_historical: false
+    keyword: YEARS
+    get_num: -32676
+  - name: FOPR
+    unit: SM3/DAY
+    is_total: false
+    is_rate: true
+    is_historical: false
+    keyword: FOPR
+    get_num: 0
+  - name: FOPRH
+    unit: SM3/DAY
+    is_total: false
+    is_rate: true
+    is_historical: true
+    keyword: FOPRH
+    get_num: 0
+  - name: FGPR
+    unit: SM3/DAY
+    is_total: false
+    is_rate: true
+    is_historical: false
+    keyword: FGPR
+    get_num: 0
+  - name: RPR:20
+    unit: BARSA
+    is_total: false
+    is_rate: false
+    is_historical: false
+    keyword: RPR
+    get_num: 20
+  - name: GOPR:RFT
+    unit: SM3/DAY
+    is_total: false
+    is_rate: true
+    is_historical: false
+    keyword: GOPR
+    wgname: RFT
+    get_num: 1
+  - name: WOPRH:R_A2
+    unit: SM3/DAY
+    is_total: false
+    is_rate: true
+    is_historical: true
+    keyword: WOPRH
+    wgname: R_A2
+    get_num: 1
```

Comment:
I am not sure we need this per keyword:well. With numerous wells, this can be very large. Could we rather have wells and groups in one list, and keywords more generic? Like:
```
data:
  wells: ["WELLA", "WELLB", ....]
  ...
  - name: WOPRH:wells            # as a generic assosiation
    ...
```
Not sure about the `keyword` entry either. Same as `name`?

Reply:
General answer: The current implementation simply extracts the metadata from .arrow and dumps them into dict. There is no domain-specific logic on it. We may want to include that at some point, but this PR does not do that. So right now, the code has no concept of e.g. `keyword` or other entries. It takes what it can find, and forwards everything to the outgoing metadata.